modules/ROOT/pages/x12-identity-settings.adoc

```diff
@@ -32,78 +30,74 @@ Valid values::
 * `SN (Standard Address Number)`
 * `ZZ (Mutually Defined)`
 
+|Yes
 
 |Identifier value
 |EDI X12-ISA ID for your trading partner; can be up to 15 characters. Your trading partner should supply you with the proper EDI X12 Interchange Sender and Receiver IDs.
 
-Required: Yes +
+|Yes +
 
 |===
 
 == X12 - GS Values
 
 Refer to this information specifically for using GS with X12.
 
-[%header,cols="3s,7a"]
+[%header%autowidth.spread]
 |===
-|Setting |Description
+|Setting |Description |Required
 
 |Identifier Value
 |EDI X12-GS ID for your trading partner; can be up to 15 characters. Your trading partner should supply you with the proper EDI X12 Group Sender/Receiver ID.
 
-Required?: Yes +
+|Yes +
 
 |===
 
 == DUNS
 
 Refer to this information specifically for using the DUNS number as an identifier.
 
-[%header,cols="3s,7a"]
+[%header%autowidth.spread]
 |===
-|Setting |Description
+|Setting |Description |Required
 
 |Identifier Value
 |The DUNS number for your trading partner. The DUNS number is a nine-digit number that your trading partner needs to supply you with that corresponds to the Sender/Receiver ID.
 
-Required: Yes +
+|Yes +
 
 |===
 
 == AS2 Identity
 
 Refer to this information specifically for using the AS2 identity to identify a trading partner.
 
-[%header,cols="3s,7a"]
+[%header%autowidth.spread]
 |===
-|Setting |Description
+|Setting |Description |Required
 
 |Identifier Value
 |The AS2 identity for your trading partner. Your trading partner should supply you with corresponding to the Sender / Receiver AS2 identity.
 
-Required: Yes +
+|Yes +
 
 |===
 
 == Delete Identifiers
 
 You might want to delete an identifier when it is no longer needed.
-Consider that deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner.
-Ask yourself the following when deleting an identifier: Why do you want to delete it? Do you need to replace it with another identifier?
-Remember that there must always be at least one identifier per message flow.
-
-After successfully deleting an identifier, you can create a new identifier with the same value as the one you deleted.
+Deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner. Remember that there must always be at least one identifier per message flow.
```

Comment:
Remember that there must..passive sentence?

You must always configure one identifier per message flow?